packages/react-router/__tests__/server-runtime/data-test.ts

```diff
@@ -1,6 +1,6 @@
 import { decodeViaTurboStream } from "../../lib/dom/ssr/single-fetch";
-import type { ServerBuild } from "../../lib/server-runtime/build";
 import { createRequestHandler } from "../../lib/server-runtime/server";
+import { mockServerBuild } from "./utils";
```

Comment:
Updated tests to use this existing utility which now includes the `routeDiscovery` config